src/Providers/Stripe/Customer/Service.php

```diff
@@ -64,17 +64,17 @@ public function find(string $uid): Customer
     /**
      * Associates a card to a customer using the Stripe SDK.
      *
-     * @param Customer $customer
-     * @param Card $card
+     * @param string $customerUid
+     * @param string $cardUid
      *
      * @return Card
      */
-    public function addCard(Customer $customer, Card $card): Card
+    public function addCard(string $customerUid, string $cardUid): Card
     {
         /** @var PaymentMethod $paymentMethod */
-        $paymentMethod = PaymentMethod::retrieve($card->uid);
+        $paymentMethod = PaymentMethod::retrieve($cardUid);
```

Comment:
I wouldn't mind reading `\Stripe\PaymentMethod::retrieve()` instead of `PaymentMethod::retrieve()` for clarity. This way I can see straight away that this is not coming from our package but from Stripe SDK. Would also prevent us having to define aliases when conflicting with out stuff.

Reply:
On our package we are using Card and not PaymentMethod. All the naming have already been discussed but happy to have a dev meeting if needed.